src/main/resources/application.yml

```diff
@@ -48,6 +54,9 @@ idam:
     totp_secret: "AAAAAAAAAAAAAAAA"
     microservice: "cmc_claim_store"
 
+pdf-service:
+  url: "${PDF_SERVICE_URL:}"
```

Comment:
shouldn't this be done automatically?
I would have thought it would convert to that